src/app/components/Tokens/TokenTransfers.tsx

```diff
@@ -63,17 +65,62 @@ const DelayedEventBalance: FC<
     }
   }, [ethAddress])
 
-  const tokenQuery = useGetRuntimeEvmTokensAddress(event.network, event.layer, oasisAddress, {
-    query: {
-      enabled: !!oasisAddress,
+  const { isLoading, isError, data } = useGetRuntimeEvmTokensAddress(
+    event.network,
+    event.layer,
+    oasisAddress,
+    {
+      query: {
+        enabled: !!oasisAddress,
+      },
     },
-  })
+  )
+  const token = data?.data
+
+  if (isLoading) {
+    return <Skeleton variant="text" />
+  }
+  if (isError || !token) {
+    return t('common.missing')
```

Comment:
Hm, in case of a different type of transfer we dont know where amount field is, so maybe throw in devmode and "n/a" in production

Reply:
OK, throwing in dev mode, "n/a" in production.

Reply:
Can you add snapshot tests for all these edge cases

Reply:
Resurrecting with an example:
https://explorer.dev.oasis.io/mainnet/emerald/address/0xceF2f95f185D49bcd1c10DE7f23BEaCBaae6eD0f/token-transfers#transfers

![image](https://github.com/oasisprotocol/explorer/assets/3758846/6a55ab7b-dd88-4f25-999f-96656a8b26b6)

instead of n/a we could show the raw transfer amounts
```js
"value": "1004350000000000000"
"value": "24224120000000000000"
"value": "895423197350738460735"
"value": "107180000000000000"
```